scripts/logging-functions.js

```diff
@@ -0,0 +1,4 @@
+const test = (inputValue1, inputValue2) => {
+  return 3 * inputValue1 + 4 * inputValue2;
```

Comment:
One more change here. Lol. Use parentheses (brackets) so that the return value can be clearer to a person reading your code.  `(3 * inputValue1) + (4 * inputValue2);`

Reply:
okay.. Do i need to close this merge request and initiate another one for my the new commits to take effect on my develop branch?

Reply:
No. When you make the next commit based on the above, then the pull request will be ready for merging. After that, I'd give a go-ahead to merge to develop.

Reply:
Done!